services/crates/crates-downloads.service.js

```diff
@@ -9,49 +9,54 @@ export default class CratesDownloads extends BaseCratesService {
     pattern: ':variant(d|dv|dr)/:crate/:version?',
```

Comment:
For the jsdelivr badge you leveraged a period enum, whereas here you broke down wvery example. Any specific reason why?

Reply:
Yes. I think the variants here need more explanation.

I don't think I would know

d = total downloads
dv = downloads for version
dr = recent downloads

so I think it makes sense to give each one an individual title.

In the jsdelivr example, it was more straightforward to just explain this in the param description: `description: 'Hits per day, week, month or year',`.

Reply:
Thanks for explaining, makes sense to me.